prompts/all.json

```diff
@@ -108,7 +119,7 @@
         "- City: city to be shipped",
         "- Country: country of city"
       ]
-    }, 
+    },
     {
       "id": "input-output-pairs",
       "label": "Create question and answer data based on a source text",
```

Comment:
Can you remove this duplicate question-answer prompt?

Reply:
Will do. It didn't occur to me that there might already be one.